reputation/peer.go

```diff
@@ -48,23 +48,32 @@ func (x PeerID) WriteToV2(m *reputation.PeerID) {
 //
 // Argument MUST NOT be mutated, make a copy first.
 //
-// See also CompareKey.
+// Be noticed, this is a compressed public key. See [elliptic.MarshalCompressed].
+// Use [neofsecdsa.PublicKey.Decode] to get public key back.
+//
+// See also [ComparePeerKey].
 func (x *PeerID) SetPublicKey(key []byte) {
 	x.m.SetPublicKey(key)
 }
 
-// PublicKey return public key set using SetPublicKey.
+// PublicKey return public key set using [PeerID.SetPublicKey].
 //
 // Zero PeerID has zero key which is incorrect according to NeoFS API
 // protocol.
 //
+// Be noticed, this is a compressed public key. See [elliptic.MarshalCompressed].
+// Use [neofsecdsa.PublicKey.Decode] to get public key back.
+//
 // Return value MUST NOT be mutated, make a copy first.
 func (x PeerID) PublicKey() []byte {
 	return x.m.GetPublicKey()
 }
 
 // ComparePeerKey checks if the given PeerID corresponds to the party
 // authenticated by the given binary public key.
+//
+// Be noticed, this is a compressed public key. See [elliptic.MarshalCompressed].
+// Use [neofsecdsa.PublicKey.Decode] to get public key back.
```

Comment:
It takes `key` as a parameter, so the decoding is likely not relevant at all while the text can be a bit more tailored to this case.

Reply:
Yeah, updated text